modules/spacelift-space/README.md

```diff
@@ -0,0 +1,119 @@
+# Component: `spacelift-space`
+
+This component is responsible for provisioning Spacelift spaces.
+
+## Usage
+
+**Stack Level**: Global
+
+NOTE: The input `labels` will be applied to every space. To overwrite (not append) the `labels` key can be used per space as well.
+
+```yaml
+# stacks/catalog/spacelift/space/defaults.yaml
+components:
+  terraform:
+    spacelift-space/defaults:
+      metadata:
+        component: spacelift-space
```

Comment:
I was pretty sure the metadata portion is inherited since thats how a lot of the spacelift policy and spacelift infrastructure stuff is working for where this component is downstreamed. I'll double check

Reply:
Good call on the abstract. I'll fix this